src/libopensc/reader-pcsc.c

```diff
@@ -1221,7 +1222,7 @@ static void detect_reader_features(sc_reader_t *reader, SCARDHANDLE card_handle)
 	}
 
 	/* Detect display */
-	if (priv->pin_properties_ioctl) {
+	if (priv->pin_properties_ioctl && gpriv->SCardControl) {
```

Comment:
nit. In some cases in this commit you test `gpriv->SCardControl` and in some `gpriv->SCardControl != NULL`. Its usually consistent with the surrounding code, but not consistent together. Would be great to stick to either of these. I dont have strong opinion on which, but likely the first one as I see it as more commonly used and shorter.

Reply:
I was looking more on the surrounding code. I'll see what I can do